core/storage/statistics/gae_models.py

```diff
@@ -1615,12 +1791,13 @@ def get_model_instance(
         return None
```

Comment:
Btw, here the code can be simplified to just `return cls.get(instance_id, strict=False)` without the change in functionality.

Reply:
Done